scripts/qemu/run-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright (C) 2023 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -x
+set -e
+
+repo=$1
+branch=$2
+
+echo password | sudo -Sk apt update
+echo password | sudo -Sk apt install -y git cmake gcc g++ numactl hwloc libhwloc-dev libnuma-dev
+
+numactl -H
+
+git clone $repo umf
+cd umf
+git checkout $branch
+
+mkdir build
+cd build
+
+cmake .. \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -DUMF_BUILD_OS_MEMORY_PROVIDER=ON \
+    -DUMF_ENABLE_POOL_TRACKING=ON \
+    -DUMF_FORMAT_CODE_STYLE=OFF \
+    -DUMF_DEVELOPER_MODE=ON \
+    -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON \
+    -DUMF_BUILD_EXAMPLES=ON \
+    -DUMF_BUILD_LEVEL_ZERO_PROVIDER=OFF
+
```

Comment:
can you also enable pool_scalable and pool_tracking?

Reply:
mea culpa, yeah. Thx for pool scalable. Now, we have to install `libtbb-dev`, though.

Reply:
Right, done.